systemtest/src/main/java/io/strimzi/systemtest/resources/operator/BundleResource.java

```diff
@@ -200,7 +200,10 @@ public DeploymentBuilder buildBundleDeployment() {
                     envVar.setValue(Long.toString(operationTimeout));
                     break;
                 case "STRIMZI_FEATURE_GATES":
-                    envVar.setValue(Environment.STRIMZI_FEATURE_GATES);
+                    if (!envVar.getValue().equals(Environment.STRIMZI_FEATURE_GATES)) {
+                        envVar.setValue(Environment.STRIMZI_FEATURE_GATES);
+                    }
```

Comment:
I would add here to:
```java
if (!envVar.getValue().equals(Environment.STRIMZI_FEATURE_GATES)) {
   envVar.setValue(Environment.STRIMZI_FEATURE_GATES);
} else {
    # or warn 
    LOGGER.info("Not overriding FEATURE_GATES; already set in deployment.");
}

Reply:
is it a really good idea? I mean there is no logger input in builder bundle class.